public/app/plugins/datasource/loki/components/AnnotationsQueryEditor.tsx

```diff
@@ -57,6 +57,7 @@ export const LokiAnnotationsQueryEditor = memo(function LokiAnnotationQueryEdito
           onChange={onChangeQuery}
           onRunQuery={() => {}}
           history={history}
+          setQueryStats={() => {}}
```

Comment:
I would definitely make it an optional prop, instead of passing down an empty function.

Reply:
+1 on this. I know I asked why is it optional, but I didn't realize that it would have to be mocked at bunch of places. 🙂

Reply:
I was more-less just curious if you considered if it should be optional/required. 😇

Reply:
my bad, i misinterpreted the comment. ill update now